projects/ng2-konva/src/lib/components/stage.component.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @angular-eslint/no-output-native */
+import {
+  Component,
+  Input,
+  Output,
+  EventEmitter,
+  AfterContentInit,
+  ElementRef,
+  ContentChildren,
+  QueryList,
+  OnDestroy,
+  inject,
+} from '@angular/core';
+import { CoreShapeComponent as CoreShape } from './core-shape.component';
+import { updatePicture, createListener, applyNodeProps } from '../utils/index';
+import { KonvaComponent } from '../interfaces/ko-component.interface';
+import { Stage } from 'konva/lib/Stage';
+import { Layer } from 'konva/lib/Layer';
+import { ContainerConfig } from 'konva/lib/Container';
+import { NgKonvaEventObject } from '../interfaces/ngKonvaEventObject';
+import { PropsType } from '../utils/types';
+
+@Component({
+  selector: 'ko-stage',
+  standalone: true,
+  template: `<div><ng-content></ng-content></div>`,
+})
+export class StageComponent
+  implements KonvaComponent, AfterContentInit, OnDestroy
+{
+  private nodeContainer = inject(ElementRef).nativeElement;
```

Comment:
This line gives me the error `VM1600:1 ERROR Error: NG0203: inject() must be called from an injection context such as a constructor, a factory function, a field initializer, or a function used with `runInInjectionContext`. Find more at https://angular.io/errors/NG0203
    at injectInjectorOnly (core.mjs:645:15)
    at ɵɵinject (core.mjs:656:60)
    at inject (core.mjs:740:12)
    at new StageComponent (stage.component.ts:31:33)
    at NodeInjectorFactory.StageComponent_Factory [as factory] (stage.component.ts:28:28)` when switching to the standard builder `@angular-devkit/build-angular:browser.`

Reply:
I just created a completely new app with angular 16 with `ng new NAME`, installed ng2-konva and konva, copied over one example and had no issues.
My new app is using `@angular-devkit/build-angular:browser` builder.
I'm pretty sure the code in the StageComponent is correct.
I can only imagine a misconfiguration or something on your site. If you can give me more info on how to replicate the issue I can try to help fix it.